Project/ProjectSettings/ProjectSettings.asset

```diff
@@ -3,7 +3,7 @@
 --- !u!129 &1
 PlayerSettings:
   m_ObjectHideFlags: 0
-  serializedVersion: 18
+  serializedVersion: 15
```

Comment:
What version of Unity did you use? I don't think the serialization version should have changed.

Reply:
2018.4.9f1.  I will update to the latest `2018.4.x` once I am near a faster internet connection and re-generate this change.

Reply:
No need, I think `2018.4.9f1` works